lock-impl/src/main/java/com/palantir/lock/logger/SimpleLockRequestsWithSameDescriptor.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.lock.logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SimpleLockRequestsWithSameDescriptor {
```

Comment:
This class appears to have some unused methods.  Wondering if we should be using a Pair instead since this doesn't appear to add any functionality.